### ISSUE TITLE
User Content Link Tracking: clicking link from Gmail sometime counts as 2 clicks

### ISSUE THREAD

**Issue by JuanLucha** [Automattic/wp-calypso#82588]
We noticed that clicking on a user content link from Gmail sometimes results in 2 hits in the stats and sometimes only 1. We did tests with other email clients, and none gave us this behavior. This could be related to Google doing some link checks in the background, but we need to investigate more.

**Comment by TimBroddin** [comment]
Fun way to test this out:

- Make a Node.js express app that logs the full `req` object to a file/screen
- Setup up an ngrok tunnel
- Mail yourself a link to this tunnel
- See what happens

If this request is made with a specific user agent, exclude it from inserting a row

**Comment by JuanLucha** [comment]
Good idea @TimBroddin !
I have set up such an express application and wired it to the internet with a Ngrok tunnel. These are my findings:
- The server was hit twice when I was editing the post, but that's normal as Guttenberg tries to fetch the content and show a preview or integration in some cases.
<img width="807" alt="Screenshot 2023-11-28 at 11 10 26" src="https://github.com/Automattic/wp-calypso/assets/3832570/bc8b086e-f3d8-4963-8efc-e98c90b0236c">

- Then, when I opened the email and clicked the link in Gmail, only one hit to my server was done:
<img width="865" alt="Screenshot 2023-11-28 at 11 11 13" src="https://github.com/Automattic/wp-calypso/assets/3832570/c9ba6529-8295-416f-a816-d1bfc1b8bec7">

- Then, it shows only one click on the stats for that post:
<img width="1037" alt="Screenshot 2023-11-28 at 11 13 03" src="https://github.com/Automattic/wp-calypso/assets/3832570/9f68be38-2961-4208-ab06-3bcb50aa1983">

So, in my tests from yesterday, all is okay; there is no double click or anything. I was subscribed to this blog four times, so when I opened the email in Google in my inbox, they opened as a chain. That's why it says "4 open", which is again true.

I'll close this as not reproducible anymore.
